extensions/wikia/FacebookClient/scripts/preferences.js

```diff
@@ -134,6 +144,42 @@
 			window.GlobalNotification.show(msg, 'error');
 		}
 
+		function facebookError() {
+			$(document).on('tab-fbconnect-prefstext-complete', function () {
+				$('.tab-fbconnect-prefstext').on('click', function () {
+					var $tabContentContainer = $('#mw-prefsection-fbconnect-prefstext');
+
+					// Disable everything within the tab
+					$tabContentContainer
+						.find('input, button')
+						.attr('disabled', 'disabled');
+					$tabContentContainer
+						.find('a')
+						.css('pointer-events', 'none');
+
+					// Throw an error message up
+					function createModal(uiModal) {
+						var modalConfig = {
+							vars: {
+								id: 'FbErrorModal',
```

Comment:
The naming convention is actually lowerCamelCase for ids https://github.com/Wikia/guidelines/blob/master/SCSS/CodingConventions.md#class-and-id-naming-conventions